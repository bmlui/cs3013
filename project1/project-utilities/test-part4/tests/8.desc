more than one input file (one file does not exist)
