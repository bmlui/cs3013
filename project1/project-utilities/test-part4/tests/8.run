./wgrep Meh tests/8a.in tests/8b.in tests/8c.in
