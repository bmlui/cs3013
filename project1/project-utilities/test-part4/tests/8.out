k FAGvKDQhYUxaRaYDOLGeCWnooCkAJuTaZxmlfcBZvOYEFMehuiFXmIHRGH
JIvdXVFMcszCMehLOwDpzIStEJlQpeTuYhHBrprSVxvHqutvvLAaOQXuZCGO
WgMeh hfxQpfMwddNhKTxEqlAqbNMybnByCAfdSZRgfQorTuUThqCxNfEjxM
PgMtjlSOjfqwaMehAMOpXxZNKfLMvIbBJrBARHlHJbPCILteTtiVneXaKvHI
xLRhignjFLTDqapWANAbAgdTgrrDSYVoJntRnCHaJzLCMehUouaSVoTHrVrY
aioOEInOJjEsxgJuNUPxDijSORWYiDUMuGHGamMehHqzgvmMGvcgEykRVLui
lnnTlfbdmNioE SfacyoXWWTVMehaZWrjEZzEyGAN rwqNYhqUgBaeUXlNTk
GjhyQhFLMVeYmsDpgjTzwyzZJzVPYIYEMxMPsjMehhJeZVmYacJURtjgAjnZ
sPuPazqVGZfStPcedyTvarJMSNaf QFkBFtozQoZCANPZAXVoMehgYsPKFWe
RyY VqJwERdJyfKrVbcizMehTeFPCTJoQNFDzRbNgDjuhzBsIluBLHOwJqYJ
UgOdKIJAAWMZjHrZCZdcyToTUUYMbZqjKnCsaUNjeXulJZEXuDTsvvvJMeha
gRAaFP kIEEWQgdVeljcvDNeuUpUZg rsQhMehNQGnaQsFZdefhkXtuhYytl
dUjhivXJcWjCvStsZTOAEQlapCHPjdoHiUTMehfIwveIdreGHxweTyaOonXR
VScaOKJEiS NzpdvusYuvmCLneqSDGrrMehATfqHwqCrac CAMXrRIprsQSl
gzhwoxsdztaQzceYaxqBXnGQbWxeiELKRcyshvoXdUXNIRsQKaWMehlUkJIr
oZFyWJOMZMIYmsPMehfWX IulEG NfCLefikVbmxoNCMwmyPWttASHDTVvra
hgFEcMehgzMCBRFmwtotJlVWAnPqaKZqNZdKAwWbsBezqRPeXqCPqTYJZXKx
AaFkByZrJafXmmOHPrVOrPCMehCkCWDuHmcIzHaRZfeBJOOEAYfvirkSxNbs
MehSdsJSlIUmiZIWpAhTRxVbeydfmEqlRGnNXFBPKsyDDtRbTTkTxSOdxokR
XKjL gVsSGiCfQrBCdEpxyIyzrUQEjMqzLjLArRZfMQtMehGsWhCNKPosizp
BHxQHwNpYJQUCCCrY GzsymctOKuujQNYNAyGrdggxUlNAtXwe fMehRsJrU
ZjHNeGaCvtyxminbNnozZrfZWuVoWAeuBUMehWUAToqegBqkeScQFeKpjyZ
prfPZV zxxooKuOhVhMZejvrjNPsTucsWhMehdFTNXNIhkctZZslEpsDZBOr
 otjzwiKmoUwWKLqqLUXzbLUmaUZdGXuDMehfvHbdRjIVvENYrpBofhBTBoW
aahMehsCymDhwjBuGIVAm kroAQfBzjlJ wtwEgUjgTYqlZDXYIHjDXzVEPJ
AYsbgYJxCJnfGVsuGgvTDynLuLtInVapxPbYV BYItZaYmLLhLCMehJlgbCV
lMehFr ROOwnfbNZhA vpfViCZpYXuWZsdKFtEdTBDdCtlwv MXQtBaqbKTG
ZkQlcMehvWkkpTGZGCqgoeotjxqlabgECXxcQSKtFMpIFkVEFCrksZJzKCSe
XSeERHqrAXoquGcjNxIQVbnrMehjOxoSBcClxRdipPzTyYfwdmZDA mbFdwk
wgrep: cannot open file
